docs/manage/CLI.md

```diff
@@ -152,6 +159,43 @@ To switch to using an organization API key for authentication right away, [logou
 
 An organization can have multiple API keys.
 
+### Create a location API key
+
+To use an location API key to authenticate your CLI session, you must first create one:
+
+1. First, [authenticate](#authenticate) your CLI session.
+   If you don't already have a location API key created, authenticate using a personal access token or an organization API key.
+
+1. Then, run the following command to create a new location API key:
+
+   ```sh {class="command-line" data-prompt="$"}
+   viam locations api-key create --location-id <location-id>
+    --org-id <org-id> --name <key-name>
+   ```
+
+   Where:
+
+   - `location-id` is your location ID.
+     You can find your location ID by running `viam locations list` or by visiting your [robot fleet's page](https://app.viam.com/robots) in the Viam app.
+   - `org-id` is an optional organization ID to attach the key to.
+     You can find your organization ID by running `viam organizations list` or by visiting your organization's **Settings** page in [the Viam app](https://app.viam.com/).
+     If omitted, the key will attempt to attach itself to the organization of the location if there is only one organization associated with that location.
+   - `key-name` is an optional name for your API key.
+     If omitted, a name will be auto-generated based on your login info and the current time.
+
+The command will return a `key id` and a `key value`.
+You will need both to authenticate using `viam login api-key`.
+
+{{% alert title="Important" color="note" %}}
+Secure these key values safely.
+Authenticating using a location API key gives the authenticated CLI session full read and write access to all robots in that location.
+{{% /alert %}}
+
+Once created, you can then use the location API key to authenticate future CLI sessions.
```

Comment:
This will merge today, and then the link will work once you rebase
```suggestion
Once created, you can use the location API key to authenticate future CLI sessions or to [connect to robots with the SDK](/program/#authenticate).
```